### PR TITLE
github oauth v3

### PR DESCRIPTION
**Pull request by jmarshak**
implemented https://developer.github.com/v3/auth/

### CODE REVIEW

File: drivers/github/index.js [hunk 1]
```diff
@@ -13,11 +13,15 @@ module.exports =  {
...
 function createRepo(options) {
     let githubConfig = _.find(config.get().drivers, { type: 'github' });
-    const url = `https://api.github.com/user/repos?access_token=${githubConfig.accessToken}`;
+    const url = `https://api.github.com/user/repos`;
     return axios.post(url, {
         name: options.name,
         description: "created by bitcar",
         private: options.private || false
+    }, {
+        headers: {
```
Comment (bdite) [inline drivers/github/index.js]: Have you been able to test what happens if there isn't an access token?

Comment (jmarshak) [reply drivers/github/index.js]: I forgot about that, will do.

Comment (jmarshak) [reply drivers/github/index.js]: done.

**Review by bdite — APPROVED** [review]

**Comment by bdite** [comment]
BUMP

**Review by machellerogden — APPROVED** [review]
Thank you! I've neglected this project for a very long time even though I still use it everyday. Today I decided it was time to patch the auth issue and found your PR waiting for me. Really appreciate it!

**Comment by machellerogden** [comment]
v1.7.4 is now published.

**Comment by machellerogden** [comment]
@jmarshak thanks again

**Comment by jmarshak** [comment]
Awesome!

**Comment by bdite** [comment]
Hooray!!

**Comment by skanjo** [comment]
thanks @jmarshak

**Pull request merged**